brigadier/home/views.py

```diff
@@ -1,8 +1,78 @@
 from django.views.generic import TemplateView
+from django.utils import timezone
+from django.db.models import Count
+
+from projects.models import Project, Task, IN_PROGRESS, COMPLETED
+from employees.models import Employee
 
 
 class HomeView(TemplateView):
     """View of the home page.
 
     """
     template_name = 'home.html'
+    context_object_name = 'statistic_lists'
+
+    def get_context_data(self, **kwargs):
+        """Get context data for home page view:
+            projects_statistics - statistics of the projects
+            tasks_statistics - statistics of the Tasks
+            employees_statistics - statistics of the Employees
+
+        """
+        projects_statistics = {
+            'total': Project.objects.count(),
+            'in_progress': Project.objects.filter(closed=False).count(),
+            'overdue': Project.objects.filter(
+                deadline__lt=timezone.now()
+            ).count(),
+            'concluded': Project.objects.filter(closed=True).count(),
```

Comment:
it's for queries, this is a performance 4x overhead
it's supposed to be a one grouping query

Reply:
what concluded means?